src/Motion.js

```diff
@@ -217,6 +260,12 @@ function shouldStopAnimation (currentValues, values, currentVelocities) {
     // istanbul ignore if
     if (!Object.prototype.hasOwnProperty.call(values, key)) continue
 
+    if (isArray(values[key]) || isObject(values[key])) {
+      if (shouldStopAnimation(currentValues[key], values[key], currentVelocities[key])) {
```

Comment:
Mmh, yeah, I'd have done it the other way around. I'll test locally. It's also better to skip the other tests, I think